dashboardgroup.go

```diff
@@ -29,6 +29,10 @@ func (c *Client) CreateDashboardGroup(dashboardGroupRequest *dashboard_group.Cre
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
+		return nil, errors.New("Unexpected status code: " + resp.Status)
```

Comment:
Consider whether the response body might be useful to dump into the error message as well.  I guess it depends on what kinds of input might trigger a non-200 response and how helpful the response is in diagnosing it.

Reply:
No reason to not be consistent, though. Excellent point! Will add and merge.